docs/framework/configure-apps/file-schema/network/defaultproxy-element-network-settings.md

```diff
@@ -19,13 +19,12 @@ Configures the Hypertext Transfer Protocol (HTTP) proxy server.
 ## Syntax  
   
 ```xml  
-      <defaultProxy  
-        enabled="true|false"  
-        useDefaultCredentials="true|false">  
-           <bypasslist> … </bypasslist>  
-           <proxy> … </proxy>  
-           <module> … </module>  
-      </defaultProxy>
+<defaultProxy enabled="true|false"  
```

Comment:
Can you please use original indentation (4 spaces for subelements and 2 spaces for attributes) and line wrapping? (multi-line attributes start on new line) I assume it is consistent with other docs.

(applies to multiple occurrences later on as well)

Reply:
On the contrary, I think my changed styles are consistent with other docs.

See also #9852

Reply:
According to the default behaviors of `XmlWriterSettings.NewLineOnAttributes`, it could be better to use 2 spaces indentation for both subelements and attributes, and start a new line for all attributes if there are any multi-line ones.

Reply:
I would expect a consensus since there are unformatted blocks elsewhere that need to be fixed.

Reply:
That should be separate discussion IMO.